CHANGELOG.md

```diff
@@ -35,21 +35,28 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
-## [v10.0.0-rc1] - 2022-11-04
+## [v10.0.0-rc2] - 2022-12-04
 
 ### State Machine Breaking
 
+- (deps) [\#1140](https://github.com/evmos/evmos/pull/1140) Bump Ethermint version to [`v0.20.0-rc3`](https://github.com/evmos/ethermint/releases/tag/v0.20.0-rc3)
 - (ante) [#1054](https://github.com/evmos/evmos/pull/1054) Remove validator commission `AnteHandler` decorator and replace it with the new `MinCommissionRate` staking parameter.
-- (deps) [\#1041](https://github.com/evmos/evmos/pull/1041) Add ics23 dragonberry replace in go.mod as mentioned in the [Cosmos SDK release](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.4)
-- (deps) [\#1037](https://github.com/evmos/evmos/pull/1037) Bump Ethermint version to [`v0.20.0-rc2`](https://github.com/evmos/ethermint/releases/tag/v0.20.0-rc2)
+- (deps) [\#1041](https://github.com/evmos/evmos/pull/1041) Add ICS-23 Dragonberry replace in `go.mod` as mentioned in the [Cosmos SDK release](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.4)
 - (vesting) [\#1070](https://github.com/evmos/evmos/pull/1070) Add Amino encoding support to the vesting module for EIP-712 signing.
 
 ### API Breaking
 
 - (erc20) [\#914](https://github.com/evmos/evmos/pull/914) Support registering multiple assets on `RegisterCoinProposal` and `RegisterERC20Proposal`
 
+### Features
```

Comment:
The difference between `Features` and `Improvements` isn't clear here. Would just keep improvements or expect e.g. to include the automated coin conversion under improvements